TODO.md

```diff
@@ -1,4 +1,5 @@
+# ToDo
```

Comment:
Do we need that file or can we delete it?

Reply:
good question - @mandelsoft is it safe to remove that TODO file? Are those topics done? Do we need to create issues for the remaining? Which ones?

Reply:
it's removed now